tools/cloud_functions/gcs_event_based_ingest/gcs_ocn_bq_ingest/README.md

```diff
@@ -60,12 +60,91 @@ DEFAULT_DESTINATION_REGEX = (
     r")?"                             # [end]yyyy/mm/dd/hh/ group (optional)
     r"(?P<batch>[\w\-_0-9]+)?/"       # batch id (optional)
 )
-`
+```
+
+## Monitoring
+### `monitored_tables`
```

Comment:
Add a note on how to deploy the monitoring scripts (e.g. snippet of setting DATASET env var and run your shell script)